### ISSUE TITLE
[Request] Clear Logs

### ISSUE THREAD

**Issue by wdahlenburg** [wdahlenburg/interactsh-collaborator#5]
Adding a feature request to add the ability to clear all logs from the table

**Comment by faisalrathore93** [comment]
yes please clear log feature is needed. Can't thankyou enough for this extension.

**Comment by faisalrathore93** [comment]
if you could please add another feature like 'sort by' time or match a word in the request or response. and a refresh button.

**Comment by iNoSec2** [comment]
+1 for the possibility to delete items, and if possible to highlight them with colors like all others tabs in burp it will be perfect!! Thanks a lot for this awesome extension
